ethpm_types/manifest.py

```diff
@@ -291,9 +291,6 @@ def add_compilers(self, *compilers: Compiler):
               compilers.
         """
 
-        full_list: List[Compiler] = self.compilers or []
-        for compiler in compilers:
-            if compiler not in full_list:
-                full_list.append(compiler)
-
-        self.compilers = full_list
+        start = self.compilers or []
+        start.extend([c for c in compilers if c not in start])
```

Comment:
for the future I think `.extend` takes in a sequence so you can just do `.extend(c for c in ...)`